schema/blocks/questionnaire.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "section_number": {
+        "type": "string"
+      },
+      "title": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "number": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "pattern": "^Questionnaire$"
```

Comment:
I think we need to raise a card for renaming this block type from Questionnaire to Question. What do you think?

Reply:
Yeh, given we not only have a single question per page this probably makes sense

Reply:
https://trello.com/c/wbHsWRLp/430-rename-block-type-from-questionnaire-to-question